examples/cpp/CMakeLists.txt

```diff
@@ -36,24 +36,24 @@ set(BUILD_CUVS_C_LIBRARY OFF)
 include(../cmake/thirdparty/get_cuvs.cmake)
 
 # -------------- compile tasks ----------------- #
-add_executable(BRUTE_FORCE_EXAMPLE src/brute_force_bitmap.cu)
-add_executable(CAGRA_EXAMPLE src/cagra_example.cu)
-add_executable(CAGRA_PERSISTENT_EXAMPLE src/cagra_persistent_example.cu)
-add_executable(DYNAMIC_BATCHING_EXAMPLE src/dynamic_batching_example.cu)
-add_executable(IVF_FLAT_EXAMPLE src/ivf_flat_example.cu)
-add_executable(IVF_PQ_EXAMPLE src/ivf_pq_example.cu)
+#add_executable(BRUTE_FORCE_EXAMPLE src/brute_force_bitmap.cu)
```

Comment:
Reminder to uncomment this. Perhaps we can add a `--build-examples` flag to `build.sh` that allows the user to specify which examples they want to build (we already have this for benchmarks, for example).

Reply:
Un-commented all of the tests, can add `--build-examples` or more convenient build script with a future PR